src/app/child-dev-project/notes/demo-data/demo-note-generator.service.ts

```diff
@@ -3,7 +3,7 @@ import { DemoDataGenerator } from "../../../core/demo-data/demo-data-generator";
 import { Injectable } from "@angular/core";
 import { Child } from "../../children/model/child";
 import { Note } from "../model/note";
-import { faker } from "../../../core/demo-data/faker";
+import faker from "faker/locale/en_IND";
```

Comment:
can we do this with an intermediate wrapper file? Would be nice to be able to change the locale with one line instead of across dozens of files.